src/assets/styles/components/_checkbox.css

```diff
@@ -0,0 +1,58 @@
+/* Custom checkboxes. Reference: https://codepen.io/spacemonkey/pen/vmZROv */
+.checkbox {
```

Comment:
Any reason to require a class rather than styling checkboxes using `[type="checkbox"]`?